packages/core/contracts/optimistic-governor/implementation/OptimisticGovernor.sol

```diff
@@ -418,9 +420,11 @@ contract OptimisticGovernor is OptimisticOracleV3CallbackRecipientInterface, Mod
 
     // Caches the address of the Optimistic Oracle V3 from the Finder.
     function _sync() internal {
-        optimisticOracleV3 = OptimisticOracleV3Interface(
-            finder.getImplementationAddress(OracleInterfaces.OptimisticOracleV3)
-        );
+        address currentOptimisticOracleV3 = finder.getImplementationAddress(OracleInterfaces.OptimisticOracleV3);
+        if (currentOptimisticOracleV3 != address(optimisticOracleV3)) {
+            optimisticOracleV3 = OptimisticOracleV3Interface(currentOptimisticOracleV3);
+            emit OptimisticOracleChanged(currentOptimisticOracleV3);
```

Comment:
this seems a bit counter intuitive to emit the `currentOptimisticOracleV3` over the `optimisticOracleV3`. would you not want to emit the value of the _new_ oov3?

Reply:
Agree, fixed in 3ee4250242d184edbcc480d128f9af55c996eb26 by renaming currentOptimisticOracleV3 to newOptimisticOracleV3.